File: The Core/4. Loop Tunnel/30. Apple Boxes/appleBoxes.cs
int appleBoxes(int k) {
    bool isOdd = true;
    int totalRed = 0;
    int totalYellow = 0;
    
    for(int i = 1; i <= k; i++){
        if(isOdd){
            totalYellow = totalYellow + (i * i);
        }
        else{
            totalRed = totalRed + (i * i);
        }
        
        isOdd = !isOdd;
    }    
    return totalRed - totalYellow;
}

File: The Core/Loop Tunnel/30. Apple Boxes/appleBoxes.py
def appleBoxes(k):
    isOdd = True
    totalRed = 0
    totalYellow = 0
    
    for x in range(1, k+1):
        if(isOdd):
            totalYellow = totalYellow + (x * x)
        else:
            totalRed = totalRed + (x * x)
        
        isOdd = not isOdd

    return totalRed - totalYellow
